docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    transported = dual_potentials.transport(x)
</pre><p>both in this example and that below, it would be great to draw a fresh sample for x, to showcase the out of sample property of the estimator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>